algowiz-app/src/components/viz/VizDijkstra.js

```diff
@@ -0,0 +1,434 @@
+import React from "react";
+import ReactFlow from "react-flow-renderer";
```

Comment:
The tests are failing because this module is not installed. Can we add this to the package.json and push again?